build.gradle

```diff
@@ -328,6 +333,7 @@ configure(opentelemetryProjects) {
                 prometheus_client_common: "io.prometheus:simpleclient_common:${prometheusVersion}",
                 protobuf                : "com.google.protobuf:protobuf-java",
                 protobuf_util           : "com.google.protobuf:protobuf-java-util",
+                weak_lock_free          : "com.blogspot.mydailyjava:weak-lock-free:${weakLockFreeVersion}",
```

Comment:
with the code vendor'd in, do we still need this?

Reply:
Oops thanks